app/src/components/grids/cardsGrid.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import React from "react";
+import useBreakpoints from "../../hooks/useBreakpoints";
+import CollectionDataType from "../../types/CollectionDataType";
+import { CollectionCardModel } from "../../models/collectionCard";
+import ItemDataType from "@/src/types/ItemDataType";
+import ItemCardDataType from "@/src/types/ItemCardDataType";
+import { ItemCardModel } from "@/src/models/itemCard";
+import { SimpleGrid as DCSimpleGrid } from "../simpleGrid/simpleGrid";
+import { Card as DCCard } from "../card/card";
+
+interface CardsGridProps {
+  records: CollectionDataType[] | ItemDataType[];
+}
+
+function isCollectionType(
+  records: CollectionDataType[] | ItemDataType[]
+): records is CollectionDataType[] {
+  return "numberOfDigitizedItems" in records[0];
+}
+
+export const CardsGrid = ({ records }: CardsGridProps) => {
```

Comment:
This looks good. It's already being used in this PR but will it also be used in the `Lane` component? This similar DOM exists there so you could use this component there.

If you do use it, however, then you might not need the type guard or `useBreakpoints` since that already happens in the `Lane`. I imagine that will break functionality in `collectionLanePage.tsx` or `divisionPage.tsx`? It's fine if you can't reuse the component in `Lane` but just wanted to point it out.

Reply:
Open to having a conversation about this but atm I would like to separate the `Lane` from this component. Conceptually they are different. Lane as a see more link and always show 4 cards,`CardsGrid` can show any amount of cards & doesn't expect to have a see more link.

Reply:
The `CardsGrid` could also be flexible enough to support four cards or x amount and conditionally show the "see more" link. But not needed right now.